tests/scattering1d/test_timefrequency_scattering.py

```diff
@@ -6,35 +6,39 @@
 from kymatio.scattering1d.core.scattering1d import scattering1d
 from kymatio.scattering1d.filter_bank import compute_temporal_support, gauss_1d
 from kymatio.scattering1d.core.timefrequency_scattering import (
-    joint_timefrequency_scattering, time_scattering_widthfirst,
-    frequency_scattering, time_averaging, frequency_averaging)
+    joint_timefrequency_scattering,
+    time_scattering_widthfirst,
+    frequency_scattering,
+    time_averaging,
+    frequency_averaging,
+)
 from kymatio.scattering1d.frontend.base_frontend import TimeFrequencyScatteringBase
 from kymatio.scattering1d.frontend.torch_frontend import TimeFrequencyScatteringTorch
 from kymatio.scattering1d.frontend.numpy_frontend import TimeFrequencyScatteringNumPy
 
 
 def test_jtfs_build():
     # Test __init__
-    jtfs = TimeFrequencyScatteringBase(
-        J=10, J_fr=3, shape=4096, Q=8, backend='torch')
+    jtfs = TimeFrequencyScatteringBase(J=10, J_fr=3, shape=4096, Q=8, backend="torch")
     assert jtfs.F is None
 
     # Test Q_fr
     jtfs.build()
     assert jtfs.Q_fr == (1,)
 
     # Test parse_T(self.F, self.J_fr, N_input_fr, T_alias='F')
-    assert jtfs.F == (2 ** jtfs.J_fr)
+    assert jtfs.F == (2**jtfs.J_fr)
     assert jtfs.log2_F == jtfs.J_fr
 
     # Test that frequential padding is sufficient
     N_input_fr = len(jtfs.psi1_f)
     assert (jtfs._N_padded_fr - N_input_fr) > (2 * jtfs.F)
-    assert (jtfs._N_padded_fr - N_input_fr)  > (2 * (2 ** jtfs.J_fr))
-    phi_f = gauss_1d(2 * jtfs._N_padded_fr, jtfs.sigma0/jtfs.F)
+    assert (jtfs._N_padded_fr - N_input_fr) > (2 * (2**jtfs.J_fr))
```

Comment:
is this necessary? if there is a previous line checking jtfs.F = 2**jtfs.J_fr

Reply:
ah you're right. this is redundant